app/views/layouts/angular/_gtl.html.haml

```diff
@@ -0,0 +1,44 @@
+- current_model = controller.class.model if controller.class.respond_to?("model") && !controller.class.model.nil?
+- gtl_type_string = j_str(@gtl_type)
+#miq-gtl-view{"ng-controller" => "reportDataController as dataCtrl"}
+  .msg-info{"ng-show" => "dataCtrl.showMessage && dataCtrl.showMessage !== ''"}
+    = render(:partial => "layouts/info_msg",
+               :locals => {:message => "{{dataCtrl.showMessage}}"})
+  %miq-tile-view{"ng-if"            => "dataCtrl.gtlType === 'grid' || dataCtrl.gtlType === 'tile'",
+                 "settings"         => "dataCtrl.settings",
+                 "per-page"         => "dataCtrl.perPage",
+                 "rows"             => "dataCtrl.gtlData.rows",
+                 "on-row-click"     => "dataCtrl.onItemClicked(item, event)",
+                 "on-sort"          => "dataCtrl.onSort(headerId, isAscending)",
+                 "on-item-selected" => "dataCtrl.onItemSelect(item, isSelected)",
+                 "load-more-items"  => "dataCtrl.onLoadNext(start, perPage)",
+                 "columns"          => "dataCtrl.gtlData.cols",
+                 "type"             => "dataCtrl.gtlType === 'grid' ? 'small' : 'big'"}
+  %miq-data-table{"ng-if"            =>"dataCtrl.gtlType === 'list'",
+                  "settings"         => "dataCtrl.settings",
+                  "per-page"         => "dataCtrl.perPage",
+                  "rows"             => "dataCtrl.gtlData.rows",
+                  "on-row-click"     => "dataCtrl.onItemClicked(item, event)",
+                  "on-sort"          => "dataCtrl.onSort(headerId, isAscending)",
+                  "load-more-items"  => "dataCtrl.onLoadNext(start, perPage)",
+                  "on-item-selected" => "dataCtrl.onItemSelect(item, isSelected)",
+                  "columns"          => "dataCtrl.gtlData.cols"}
+  .no-record{"ng-show" => "!dataCtrl.settings.isLoading && dataCtrl.gtlData.rows.length === 0"}
+    = render(:partial => "layouts/info_msg",
+               :locals => {:message => _("No Records Found.")})
+:javascript
+  sendDataWithRx({unsubscribe: 'reportDataController'});
+  miq_bootstrap('#miq-gtl-view', 'ManageIQ.report_data');
+  sendDataWithRx({initController: {
+    name: 'reportDataController',
+    data: {
+      modelName: '#{CGI::escape(j_str(@display.nil? && current_model ? controller.class.model.to_s.tableize : @display))}',
```

Comment:
really want to html escape javascript escaped data? if so, use `h` instead of `CGI::escape`